enterprise_learner_portal/api/v1/views.py

```diff
@@ -55,16 +42,50 @@ def get(self, request):
             user_id=user.id,
             enterprise_customer__uuid=enterprise_customer_id,
         )
-        course_ids_for_ent_enrollments = EnterpriseCourseEnrollment.objects.filter(
+        enterprise_enrollments = EnterpriseCourseEnrollment.objects.filter(
             enterprise_customer_user=enterprise_customer_user
-        ).values_list('course_id', flat=True)
-
-        overviews = get_course_overviews(course_ids_for_ent_enrollments)
+        )
 
         data = EnterpriseCourseEnrollmentSerializer(
-            overviews,
+            enterprise_enrollments,
```

Comment:
Isn't this a deoptimization? Now we will make a query to the course overview table for every enterprise enrollment instead of batching them like the previous code was doing.

Reply:
the motivation for this was so that it was easier to serialize one enrollment's data below in what the patch function returned, and to also have access to the enterprise enrollment object itself in the serializer for the status determination. i considered passing the enterprise_enrollments as context, but it seemed awkward to do that considering this serializer is supposed to serialize "enrollments". i'm not sure what the performance implications are though... i would guess the table is indexed by course id?